src/App/MainPage.xaml.cs

```diff
@@ -15,6 +16,7 @@
 using Windows.UI.Xaml.Controls;

```

Comment:
is Windows.UI.Xaml.Controls still used? if not, you could do "using Controls = Microsoft.UI.Xaml.Controls;" i think

Reply:
That namespace is used everywhere, unfortunately

Reply:
figured.